client/src/pages/About.tsx

```diff
@@ -1,8 +1,231 @@
 import React from 'react';
-import { PageWrapper } from '../components';
+import styled, { css } from 'styled-components';
+import {
+  H1,
+  H2,
+  P,
+  PageWrapper,
+  SubTitle,
+} from '../components';
+import { COLORS } from '../constants';
+import Logo from '../imgs/logo.png';
+import erinPic from '../imgs/erinPic.png';
+import andreiPic from '../imgs/andreiPic.png';
+import jodiPic from '../imgs/jodiPic.png';
+
+const Box_Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  margin-top: 1900px;
+  gap: 15px;
+`;
+
+const FlexBoxWrapper = styled.div`
+  height: 85vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 100px;
+  margin-right: 50px;
+`;
+
+const ProjectBox = styled.div`
+  width: 999px;
+  height: 850px;
+  background-color: ${COLORS.PURPLE_LIGHT};
+  border-radius: 22px;
+  margin-top: 30px;
+  margin: 5px;
+  text-align: center;
```

Comment:
The `ProjectBox` could benefit from adding extra padding- the text looks kind of tight. Just adding `padding: 45px'` here would work! (You can change the number!)